CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/TestServerContext.java

```diff
@@ -56,4 +58,9 @@ public String getLocalIpAddress() {
         return ip;
     }
 
+    @Override
+    public String customEncode(byte[] hashBytes){
```

Comment:
What would you think about renaming this "encodeBase64" or something.  It isn't really doing anything custom.  It is just doing perfectly normal base 64 encoding...

Reply:
changed the method name